qiskit_sphinx_theme/breadcrumbs.html

```diff
@@ -1,40 +1,10 @@
-{# Support for Sphinx 1.3+ page_source_suffix, but don't break old builds. #}
-
-{% if page_source_suffix %}
-{% set suffix = page_source_suffix %}
-{% else %}
-{% set suffix = source_suffix %}
-{% endif %}
-
-{% if meta is defined and meta is not none %}
-{% set check_meta = True %}
-{% else %}
-{% set check_meta = False %}
-{% endif %}
-
-{% if check_meta and 'github_url' in meta %}
-{% set display_github = True %}
-{% endif %}
-
-{% if check_meta and 'bitbucket_url' in meta %}
-{% set display_bitbucket = True %}
-{% endif %}
-
-{% if check_meta and 'gitlab_url' in meta %}
-{% set display_gitlab = True %}
-{% endif %}
-
 <div role="navigation" aria-label="breadcrumbs navigation">
 
   <ul class="pytorch-breadcrumbs">
     {% block breadcrumbs %}
       <li>
         <a href="{{ pathto(master_doc) }}">
-          {% if theme_pytorch_project == 'tutorials' %}
-            Tutorials
-          {% else %}
```

Comment:
What was this?

Reply:
I believe this was a holdover from when the Tutorials were built using a separate process from the main package? Or it's possible it was just copied over from pytorch, because I don't remember seeing the `theme_pytorch_project` variable anywhere in our codebases.